spec/lib/spec/lib/user_comm_screener_spec.rb

```diff
@@ -0,0 +1,137 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
```

Comment:
```suggestion
```

This is no longer required as it it automatically required as part of https://github.com/discourse/discourse/blob/f1c32a3cdf4ab79ca1ab746a927938060b4762a6/.rspec#L2

Reply:
Ah will remove this from my snippet